lib/index.js

```diff
@@ -9,12 +8,44 @@ const bravado = require('bravado-core');
 const cors = require('./cors');
 const routeHandler = require('./route');
 const indexRouteHandler = require('./index-route');
+const { forEachSerialP, keys, defer } = require('omnibelt');
 
 let server;
 let logger = () => {};
 
+const findExpectedHeaders = (params) => {
+  const result = [];
+  if (params) {
+    params.forEach(function(param) {
+      if (param.in === 'header') { result.push(param.name); }
+    });
+  }
+  return result;
+};
+
+const buildApis = async (apis) => {
+  const defs = [];
+  debug('loading api definition %s', apis);
+  if (!Array.isArray(apis)) {
+    apis = [apis];
+  }
+  await forEachSerialP(async (item) => {
+    let builtApi;
+    if ('string' === typeof item) {
+      builtApi = await bravado.ApiDefinition.build(item);
+    }
```

Comment:
shouldn't there be an else here?

Reply:
Sick catch - since this isnt a promise anymore I'm not breaking out early by returning. 👍 so yeah, else is needed.